core-bundle/src/EventListener/PreviewAuthenticationListener.php

```diff
@@ -64,6 +64,9 @@ public function __invoke(RequestEvent $event): void
             return;
         }
 
+        $context = $this->router->getContext();
+        $context->setBaseUrl('');
+
```

Comment:
Should we maybe reset the base URL after using the router? Or maybe clone the context and reset to the old context afterwards?

Reply:
Good idea.

Reply:
Fixed in 7e3aa7615.